opslevel/resource_opslevel_service.go

```diff
@@ -177,14 +186,24 @@ func reconcileTags(d *schema.ResourceData, service *opslevel.Service, client *op
 }
 
 func resourceServiceCreate(d *schema.ResourceData, client *opslevel.Client) error {
+	// owner_alias is deprecated, allow user to use one or the other but not both.
+	var ownerField *opslevel.IdentifierInput
+	if d.Get("owner").(string) != "" && d.Get("owner_alias").(string) != "" {
+		return errors.New("can pass only one of: owner owner_alias")
+	} else if d.Get("owner").(string) != "" {
+		ownerField = opslevel.NewIdentifier(d.Get("owner").(string))
+	} else if d.Get("owner_alias").(string) != "" {
+		ownerField = opslevel.NewIdentifier(d.Get("owner_alias").(string))
+	}
```

Comment:
Terraform has `GetOk` method that returns if the field was set in the configuration or not - this should help make the logic a bit more readable
```suggestion
    owner, ownerUsed := d.GetOk("owner")
    ownerAlias, ownerAliasUsed := d.GetOk("owner_alias")
	if ownerUsed && ownerAliasUsed {
		return errors.New("can pass only one of: 'owner' or 'owner_alias'")
	} else if ownerUsed {
		ownerField = opslevel.NewIdentifier(owner.(string))
	} else if ownerAliasUsed {
		ownerField = opslevel.NewIdentifier(ownerAlias.(string))
	}
```